### ISSUE TITLE
jdkPackager with exe type requires admin privileges

### ISSUE THREAD

**Issue by countfloyd** [sbt/sbt-native-packager#756]

**Comment by muuki88** [comment]
[This is the line](https://github.com/sbt/sbt-native-packager/blob/master/src/main/scala/com/typesafe/sbt/packager/jdkpackager/JDKPackagerAntHelper.scala#L132). Maybe we can make this configurable? @metasim 

A setting `val jdkPackagerAutoInstall = settingKey[Boolean]("jdkPackagerAutoInstall")` would be enough?

**Comment by countfloyd** [comment]
Yes, I think a setting would be great. Perhaps AutoInstall isn't the right name. Maybe jdkPackagerAdminInstall ??

**Comment by metasim** [comment]
@countfloyd Until time is found to work on this, one work around is to intercept/override the `antBuildDefn` task and modify the XML DOM before it gets written out to Ant.

**Comment by countfloyd** [comment]
Yes, that's exactly what I've done. Thanks.

**Comment by muuki88** [comment]
`jdkPackagerAdminInstall` sounds good to me.

**Comment by metasim** [comment]
I've not forgotten about this one.... just slammed with work ATM. Definitely should be added to the API.
